specs/fault-proof/stage-one/bond-incentives.md

```diff
@@ -125,9 +131,58 @@ opportunity cost of locking up capital in the dispute game. While we do not expl
 these costs, we assume that the current bond rewards, based on this specification, are enough as a whole to cover
 all other costs of participation.
 
+## Game Finalization
+
+After the game is resolved, claimants must wait for the [AnchorStateRegistry's
+`isGameFinalized()`](anchor-state-registry.md#isgamefinalized) to return `true` before they can claim their bonds. This
+implies a wait period of at least the `disputeGameFinalityDelaySeconds` variable from the `OptimismPortal` contract.
+After the game is finalized, bonds can be distributed.
+
+### Bond Distribution Mode
+
+The FDG will in most cases distribute bonds to the winners of the game after it is resolved and finalized, but in
+special cases will refund the bonds to the original depositor.
+
+#### Normal Mode
+
+In normal mode, the FDG will distribute bonds to the winners of the game after it is resolved and finalized.
+
+#### Refund Mode
+
+In refund mode, the FDG will refund the bonds to the original depositor.
+
+### Game Closure
+
+The `FaultDisputeGame` contract can be closed after finalization via the `closeGame()` function.
+
+`closeGame` must do the following:
+
+1. Verify the game is resolved and finalized according to the Anchor State Registry
+2. Attempt to set this game as the new anchor game .
+3. Determine the bond distribution mode based on whether the [AnchorStateRegistry's
+   `isGameProper()`](anchor-state-registry.md#isgameproper) returns `true`.
+4. Emit a `GameClosed` event with the chosen distribution mode.
+
+### Claiming Credit
+
+After the game is closed and the bond distribution mode is determined, there is a 2-step process to claim credit. First,
```

Comment:
This is a bit misleading because claimCredit() will close the game and determine the bond distribution mode.  So you don't need to close the game before starting this two step process. Maybe just make it "There is a 2-step process..."

Reply:
Thanks, resolved in 6e1a0a0